package.json

```diff
@@ -1,20 +1,22 @@
 {
   "name": "kassy",
-  "version": "1.2.0",
+  "version": "2.0.0",
   "description": "Extensible facebook chat bot.",
   "main": "main.js",
   "dependencies": {
+    "express": "^4.13.1",
     "facebook-chat-api": "^0.2.10",
     "git-pull": "^0.1.0",
     "quarantine": "^0.1.4",
     "redwrap": "0.0.4",
     "request": "^2.58.0",
-    "require-install": "^0.1.0"
+    "require-install": "^0.1.0",
+    "body-parser": "~1.13.2"
   },
   "devDependencies": {},
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node main.js"
+    "start": "node main.js slack"
```

Comment:
We want to keep this starting in test mode unless the user manually specifies that they want facebook or slack.